src/content/developers/tutorials/run-light-node-geth/index.md

```diff
@@ -1,173 +1,84 @@
+This console allows direct interaction with Ethereum. For example, running the `eth.blockNumber` command will print the latest known block number.
 
-- `eth.syncing` returns an object if your node has started but not completed block synchronization, or the value `false` if it has either completed synchronization or has not started. If the node is still synchronizing, `eth.syncing` will tell you the latest block number whose data you’ve received, as well as the total number of blocks in the current blockchain.
-- `net.peerCount` returns the number of peers you’re connected to. If this is 0, you’ll probably have to either wait a few minutes or else start Googling for solutions (could be a firewall or network issue, or something else).
-- `admin.peers` will give you a list of all peers that your node is connected to. If this is empty, then your node is not connected to any other peers.
+## Mainnet and testnets {#mainnet-and-testnets}
 
-We can also use web3js to initialize accounts, write and broadcast transaction requests to the network, look up account balances and metadata, and more. We’ll cover these operations in a later section; for now, try running the following to look up the balance of one of my accounts on the Ropsten testnet:
+Geth runs your node on [Ethereum Mainnet](/glossary/#mainnet/) by default.
 
-```js
-eth.getBalance('0x85d918c2B7F172d033D190152AEc58709Fb6D048')
-# returns 1059286000000000000 as of 11-18-2018. This value is reported in "Wei".
-# One Wei is a denomination which is equivalent to 10^-18 ether.
-# The balance of this account in ether is about 1.059eth.
+It is also possible to use Geth to run a node on one of the [public test networks](/networks/#testnets/), by running one of the following commands in Terminal:
 ```
-
-## Stopping and restarting your node {#stopping-and-restarting-your-node}
-
-You can stop your node at any time by pressing `CTRL+C`. If you want to restart the node, Geth will take a few seconds or minutes to re-synchronize (downloading the blocks and/or block headers from where it left off when the node last stopped running). If any of the above instructions aren’t working, the first thing you should do is try restarting your node.
-
-```bash
-$ geth --ropsten --syncmode "light"
+geth --syncmode light --ropsten
+geth --syncmode light --rinkeby
+geth --syncmode light --goerli
 ```
 
-Replace 'ropsten' with other testnet names as needed, or use 'mainnet'.
-
-If you are interested in running an Ethereum full node, it is generally best to do so from a dedicated machine with good network connectivity, rather than from a personal computer. Here is a guide to running a node with AWS (this is a little outdated and the referenced AMIs are no longer recent or available, so you might have to do some Googling): [How to run a node on AWS](https://medium.com/mercuryprotocol/how-to-run-an-ethereum-node-on-aws-a8774ed3acf6)
+- [Learn more about different networks](/developers/docs/networks/).
```

Comment:
```suggestion
## Where is the blockchain and EVM data stored? {#where-is-the-blockchain-and-evm-data-stored}

The directory which Geth uses to store raw blockchain data depends on your operating system. Upon running Geth, look for a message that looks like

bash
INFO [11-18|14:04:47] Allocated cache and file handles         database=/Users/bgu/Library/Ethereum/testnet/geth/lightchaindata cache=768 handles=128


The path following `“database=”` should tell you where the blockchain data is stored on your machine. If you’re running a full node, this directory will contain all of the data about every block that has ever been committed to the blockchain. Since we’re running a light node, this directory only contains the block headers.

It’s worth emphasizing here that, at the lowest level, this is where the blockchain lives. The full contents of the blockchain and the EVM state are stored on every full node in the Ethereum network, in directories that look very much like the one on your computer.

## Further reading {#further-reading}

- [Learn more about different networks](/developers/docs/networks/).
```